src/test/daml/Daml/Finance/Instrument/Generic/Test/CallableBond.daml

```diff
@@ -11,7 +11,6 @@ import DA.Map qualified as M (empty, fromList)
 import DA.Set qualified as S (empty, fromList, singleton, toList)
 import DA.Time (time)
 import Daml.Finance.Holding.Fungible qualified as Fungible (Factory(..), T)
-import Daml.Finance.Lifecycle.Election qualified as Election (Factory(..))
```

Comment:
Do you mean instead of `bootstrapBond` use `createCallableFixRateBondClaims`?

Reply:
I think that would be possible, but it would make the Generic example more complicated, because the builder uses a schedule as well. 
Also, I think the builder will get more complicated as I add more features to the dynamic callable bond.
Maybe better to keep it as is, for simplicity/clarity of the Generic example?

Reply:
agree, didn't think about the schedule, thanks!